src/utils/utils.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const getArticles = () => {
+  return axios
+    .get("https://nc-news-kx4n.onrender.com/api/articles")
+    .then(({ data }) => {
+      return data.articles;
+    });
+};
```

Comment:
Nice. Make sure to include a catch block in your useEffect so that we can add error handling later 👍